lib/authentication/SecureStorage/credentialManager.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2015-2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+const Logger = require('../../logger');
+const localStorage = require('./localStorage');
+
+/**
+ * 
+ * @param {String} host 
+ * @param {String} user 
+ * @param {String} credType 
+ * @returns 
+ */
+
+function checkForNull(...parameters){
```

Comment:
can this function be extracted to util? Seems like something that can be reused throughout project.

Reply:
Fixed.

Reply:
I changed the name 'checkForNull' to 'checkParametersDefined', and moved to the Util.js.